openapi-cli/src/main/java/io/ballerina/openapi/generators/GeneratorUtils.java

```diff
@@ -92,7 +94,11 @@
 public class GeneratorUtils {
 
     public static ImportDeclarationNode getImportDeclarationNode(String orgName, String moduleName) {
-        Token importKeyword = AbstractNodeFactory.createIdentifierToken("import ");
+        Minutiae whitespace = AbstractNodeFactory.createWhitespaceMinutiae(" ");
+        MinutiaeList leading = AbstractNodeFactory.createEmptyMinutiaeList();
+        MinutiaeList trailing = AbstractNodeFactory.createMinutiaeList(whitespace);
+
+        Token importKeyword = AbstractNodeFactory.createIdentifierToken("import", leading, trailing);
```

Comment:
Please use an util method instead of duplicating this minutiae list creation in multiple places.